.travis.yml

```diff
@@ -1,5 +1,6 @@
+# update node.js version in before_script as well, using nvm install to pick needed version
 node_js:
-  - "10"
+  - "16"
```

Comment:
This updated the version in build config (Checks --> [Travis CI](https://github.com/Netflix/lemur/runs/4306702159))
![Screen Shot 2021-11-23 at 5 21 41 PM](https://user-images.githubusercontent.com/6845690/143154093-99464a95-f80b-47e7-84fe-38bcd5d42160.png)
However travis build logs showed `node@v10.16.0 + npm@8.1.4`. Thus used nvm below to pick a node js version